plugins/guests/linux/cap/network_interfaces.rb

```diff
@@ -15,10 +20,16 @@ def self.network_interfaces(machine, path = "/sbin/ip")
             s << data if type == :stdout
           end
           ifaces = s.split("\n")
+          eth_prefix = nil
           @@logger.debug("Unsorted list: #{ifaces.inspect}")
           # Break out integers from strings and sort the arrays to provide
           # a natural sort for the interface names
           ifaces = ifaces.map do |iface|
+            unless eth_prefix
```

Comment:
I know it's Ruby, but neither @mitchellh or I are a fan of the "unless" and much prefer `if !` or `if .nil?` since it mentally maps better to the Golang folks who might look at this code in some future.